controllers/categories.controllers.js

```diff
@@ -7,3 +10,11 @@ exports.getCategories = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.getReviews = (req, res, next) => {
```

Comment:
Think about separating out getReviews - your file name is categories.controllers - maybe make a reviews.controllers.